src/templates/blog-post.tsx

```diff
@@ -91,8 +95,13 @@ const BlogHeader = ({ readingTime, frontmatter }) => {
 const BlogArticleTemplate: React.FC<BlogArticleTemplateProps> = ({
   data,
   location,
+  pageContext,
 }) => {
   const markdown = data.markdownRemark;
+  const breadcrumb = [
+    ...pageContext.breadcrumb,
+    { pathname: location.pathname, label: markdown.frontmatter.title },
+  ];
```

Comment:
I wrote the breadcrumbs in the page context because I wanted to keep the structure where all the data comes from the create-pages files. 
But I agree with you and I think that the advantage of using `t()` is much bigger and that in other places in the code the data doesn't always come from the create-pages files either. I have now adapted the code according to your review and it looks better than before. Thanks for your feedback 😊